### DISCUSSION TITLE
Other Font-Awesome Icons Don't Render

### DISCUSSION THREAD

**Discussion by neilskilling-overskilling** [devcows/hugo-universal-theme#413] [Q&A]
I am using the Hugo Universal theme and wanted to change some of the Font Awesome icons used in the features section. 

If I try and use any other Font Awesome icon other than one that has previously been used in the site it doesn't appear. I am restricting my usage to the free icons. 

For example if I change the content data/features/consulting.yaml line from: 
    `icon: "fas fa-lightbulb"`
to: 
   `icon: "fas fa-dollar-sign"`

then I just get a blank where the icon should be. Any suggestions?

**Comment by richtong** [comment]
Hmmm this seemed to work fine for me

```
icon: "fas fa-dollar-sign"
```

![image](https://github.com/user-attachments/assets/d3f6bd60-539f-4595-b6d4-f98b9ac62aa1)
